backend/src/auth/auth.controller.ts

```diff
@@ -30,7 +31,9 @@ export class AuthController {
   @ApiResponse({ status: 200, description: 'User logged in successfully.' })
   @ApiResponse({ status: 401, description: 'Invalid credentials.' })
   async login(@Body() loginDto: LoginDto, @Req() req: Request) {
-    return this.authService.login(req.user as UserDocument);
+      const user = req.user as UserDocument;
+      return this.authService.login(user);
```

Comment:
Confirmed working. I noticed that the route doesn't return the user object that you added with a previous PR, was this intended?